src/lib/components/forms/AddOnDispatch.svelte

```diff
@@ -159,28 +162,30 @@
   <slot name="premium" />
 
   <div class="controls">
-    {#if event}
-      <Button
-        type="submit"
-        mode="primary"
-        label={$_("dialog.save")}
-        disabled={disablePremium}
-      />
-    {:else}
+    {#if $me}
+      {#if event}
+        <Button
+          type="submit"
+          mode="primary"
+          label={$_("dialog.save")}
+          disabled={disablePremium}
```

Comment:
I think it'd be peferable to disable the buttons if `!$me`, instead of hiding them entirely—like you do for the BulkActions—and checking for `!$me` in the submit handler.